tests/tests_wireless_wpa3_owe_nm.yml

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# This file was generated by ensure_provider_tests.py
+---
+# set network provider and gather facts
+- hosts: all
+  name: Run playbook 'playbooks/tests_wireless_wpa3_owe.yml' with nm as provider
+  tasks:
+    - name: Set network provider to 'nm'
+      set_fact:
+        network_provider: nm
+      tags:
+        - always
+
+
+# The test requires or should run with NetworkManager, therefore it cannot run
+# on RHEL/CentOS 6
+# OWE has not been exposed via D-Bus on RHEL 7(dist-tag) and Fedora 32. Failed in setting up mock wifi on RHEL 8
```

Comment:
Same here, please specify it is not supported on `NetworkManager-1.18-8`. As @tyll said, F32 is EOL so we do not need to consider it anymore.

Reply:
Fixed.